type.proto

```diff
@@ -30,6 +30,7 @@ enum TrackingStatus {
   TRACKING = 3; // Stable tracking
   DRIFTING = 4; // Short term prediction when tracking is lost in TRACKING status
   EXPIRED = 5; // Expired tracking
+  MAX = 6;
```

Comment:
I think this can make backward compatibility issue.
If you need size of `TrackingStatus`, Can you try `TrackingStatus_ARRAYSIZE`?
Detail document is [here](https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#enum).

Reply:
Thanks! It works :smile: